history/cultures/bronzebeard.txt

```diff
@@ -74,6 +74,7 @@
 	#
 	discover_innovation = innovation_dwarven_cannon
 	discover_innovation = innovation_riflemen_dwarf
+	discover_innovation = innovation_gryphon
```

Comment:
Quote from Grox: "It wasn't until WoW that we get to see Bronzebeard or Ironforge Gryphons."
Humans and Bronzebeards shouldn't have Gryphons at the beginning of the first war! 603 or 605 imho.